### ISSUE TITLE
Don't list "closed" PRs

### ISSUE THREAD

**Issue by rdohms** [jwage/changelog-generator#57]
Right now, if a PR is rejected (i.e closed without merging) it shows up in the changelog. Since this is a change that does not active affect the code base, it feels like it should not be included in the changelog by default.

Granted we could also remove it from the milestone, but I believe it can also remain there for organization reasons, like knowing which release the PR had been intended for and finding more history on it.

**Comment by jwage** [comment]
Currently when this happens I do what you suggested and remove it from the milestone. Not opposed to adding this though if the github API supports differentiating the data.

**Comment by rdohms** [comment]
yes, so a merged PR has: state: closed and merged: true, a closed PR has merged: false.
